Support/Testing/Travis/before-script.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+##
+## Copyright (c) 2014-present, Facebook, Inc.
+## All rights reserved.
+##
+## This source code is licensed under the University of Illinois/NCSA Open
+## Source License found in the LICENSE file in the root directory of this
+## source tree. An additional grant of patent rights can be found in the
+## PATENTS file in the same directory.
+##
+
+set -eu
+
+if [[ "$TARGET" = "Android-ARM" ]]; then
```

Comment:
You should make that script take the architecture (and platform version) as a command-line option, so we should be able to install any emulator we want. You can then call this script with the right option from `install.py`.